controllers/service_accounts.go

```diff
@@ -99,7 +99,7 @@ func (r *TrustyAIServiceReconciler) createServiceAccount(ctx context.Context, in
 func (r *TrustyAIServiceReconciler) createClusterRoleBinding(ctx context.Context, instance *trustyaiopendatahubiov1alpha1.TrustyAIService, serviceAccountName string) error {
 	clusterRoleBinding := &rbacv1.ClusterRoleBinding{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      instance.Name + "-proxy-rolebinding",
+			Name:      instance.Name + "-" + instance.Namespace + "-proxy-rolebinding",
 			Namespace: instance.Namespace,
```

Comment:
nitty gritty:
clusterrolebinding does not require namespace: wont take effects

Reply:
@zdtsw I'll open a separate PR for this if that's ok with you.